pkgs/applications/misc/neo/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchurl, ncurses }:
+
+stdenv.mkDerivation rec {
+  pname = "neo";
+  version = "0.6";
+
+  src = fetchurl {
+    url =
+      "https://github.com/st3w/${pname}/releases/download/v${version}/${pname}-${version}.tar.gz";
+    sha256 = "sha256-skXLT1td4dGdsu+wbX44Z2u5sDEOKXYVVys4Q6RayIk=";
+  };
+
+  buildInputs = [ ncurses ];
+
+  meta = with lib; {
+    description = ''Simulates the digital rain from "The Matrix"'';
+    license = licenses.gpl3;
```

Comment:
gpl3Only or gpl3Plus?

Reply:
Confirmed the license, and it's GPLv3 or later.

Thanks!